nemoguardrails/library/domain_hallucination/examples.py

```
@@ -0,0 +1,262 @@
+        scoring=config.ScoringConfig(
+            fail_threshold=60.0,
+            refine_threshold=40.0,
+            warn_threshold=20.0,
+        ),
+    )
+
+    print("Current Configuration:")
+    print(json.dumps(cfg.to_dict(), indent=2))
+
+    # Save and load
+    config_path = Path("/tmp/test_config.json")
```

Ruff (0.15.20)

[error] Probable insecure usage of temporary file or directory: "/tmp/test_config.json"

(S108)